src/Response/ResponseExtensions.cs

```diff
@@ -58,5 +63,59 @@ public static class ResponseExtensions
 
             await negotiator.Handle(response.HttpContext.Request, response, obj, cancellationToken);
         }
+
+        /// <summary>
+        /// Copy a stream into the response body
+        /// </summary>
+        /// <param name="response">Current <see cref="HttpResponse"/></param>
+        /// <param name="stream">The <see cref="Stream"/> to copy from</param>
+        /// <param name="contentType">The content type for the response</param>
+        /// <param name="contentDisposition">The content disposition to allow file downloads</param>
+        /// <returns><<see cref="Task"/>/returns>
```

Comment:
Invalid XML: Move the `<see …/>` one character to the left (double `<<` and no `<` before the `/returns`)

Reply:
> A task representing the asynchronous copy to the response body.